apps/system-e2e/entrypoint.sh

```diff
@@ -14,24 +14,23 @@ if [[ "$*" =~ --project ]]; then
 fi
 
 export TEST_PROJECT TEST_ENVIRONMENT TEST_TYPE TEST_RESULTS_S3 TEST_FILTER
-
-DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+export PATH="./node_modules/.bin:$PATH"
```

Comment:
Is this smart? Might lead to opaque issues :thinking: 

However, this removes the script's dependency on `yarn`